portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -306,6 +321,14 @@ definition {
 			responseToParse = "${responseToParse}");
 	}
 
+	macro assertInFacetsWithCorrectValue {
+		var actualNumber = JSONUtil.getWithJSONPath("${responseToParse}", "$..facetValues[?(@.term=='${managerId}' && @.numberOfOccurrences==${expectedNumber})].numberOfOccurrences");
```

Comment:
please change `actual Number` to something that would reflect more what we're checking with this assert, it can be global like `actualValues`